docs/blog/newsletter-march-2019.rst

```diff
@@ -94,7 +96,7 @@ Community events coming up
 
 - 5 March - Austin, TX, USA - `Lunch meetup <https://www.meetup.com/WriteTheDocs-ATX-Meetup/events/258689786/>`_
```

Comment:
Do we only include events for the current month (March) here?

Reply:
If we include future months, I have an event to add. :)

Reply:
I've been going roughly a fortnight into the next month to make sure we don't miss anything. Send me the link and I'll add it to the list!